test/clickableText.test.js

```diff
@@ -0,0 +1,13 @@
+const test = require("tape");
+const { markdownToHtml } = require("./utils");
+const clickableText = require("../src/clickableText");
+
+test("clickable text is converted", (t) => {
```

Comment:
Might also be worth adding a control test just to verify that links without `#clickable=` aren't affected.

Reply:
Good call.  Done.